src/Components/Map.jsx

```diff
@@ -1,11 +1,12 @@
 import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
 import GoogleMapReact from 'google-map-react';
 import MapStyle from '../Modules/MapStyle'
 import { Icon } from 'semantic-ui-react';
 import Popup from 'reactjs-popup';
-import CreateImageEntry from './CreateImageEntry'
-import axios from 'axios'
-import { connect } from 'react-redux'
+import CreateImageEntry from './CreateImageEntry';
+import axios from 'axios';
+import { connect } from 'react-redux';
```

Comment:
Let's be consistent with these `;`. We either keep them or we dump them. In other Components we don't use them. Personally I vote for dumping them. But whatever we decide, we stick with it all over the app.

Reply:
Fixed!